src/JustSaying/Messaging/Middleware/Middleware`2.cs

```diff
@@ -0,0 +1,32 @@
+using System;
```

Comment:
File name and class name don't match. If the class isn't renamed, should be `MiddlewareBase(backtick)2.cs`.